docs/Advanced/Styling.md

```diff
@@ -200,6 +200,11 @@ The following rules remove the Tasks priority emoticon and render the tasks' che
 
 <!-- snippet: resources/sample_vaults/Tasks-Demo/.obsidian/snippets/tasks-plugin-priority-as-checkbox-color.css -->
 ```css
+.task-list-item[data-task-priority="highest"] input[type=checkbox] {
```

Comment:
Thank you for updating the CSS here.

The CSS content here is read from the file named on line 201 above... 
https://publish.obsidian.md/tasks-contributing/Documentation/Embedding+code+and+data+in+documentation

So please could the styles for new priorities be added to the file named on line 201? Thanks.

Oh, and also for any other CSS fragments updated elsewhere in this file...

Reply:
done!

Reply:
Thanks for fixing the first one.
Please could you also update `tasks-plugin-complete-example.css` - it needs the priority edits as well.

Here is the new version to add - just remove the .txt extension:

[tasks-plugin-complete-example.css.txt](https://github.com/obsidian-tasks-group/obsidian-tasks/files/11590784/tasks-plugin-complete-example.css.txt)

Reply:
> Thanks for fixing the first one. Please could you also update `tasks-plugin-complete-example.css` - it needs the priority edits as well.
> 
> Here is the new version to add - just remove the .txt extension:
> 
> [tasks-plugin-complete-example.css.txt](https://github.com/obsidian-tasks-group/obsidian-tasks/files/11590784/tasks-plugin-complete-example.css.txt)

Cancel that - I've got to make some other edits too, so I will do that...

Will resolve this conversation....